spec/controllers/external_users/certifications_controller_spec.rb

```diff
@@ -156,10 +156,10 @@
     context 'invalid certification' do
       it 'redirects to new' do
         params = valid_certification_params(claim, certification_type)
-        params['certification']['certification_type_id'] = 9999
+        params['certification']['certification_type_id'] = nil
         post :create, params: params
         expect(response).to render_template(:new)
-        expect(assigns(:certification).errors.full_messages).to eq(['You must select one option on this form'])
+        expect(assigns(:certification).errors.messages[:certification_type_id]).to eq(['Choose a certification type'])
```

Comment:
this should not be tested in a controller spec really - again, i know this is not you

Reply:
have applied commit for this